tests/integration/conftest.py

```diff
@@ -38,13 +38,188 @@ def tags():
 
 @pytest.fixture(scope="session")
 def resource_group():
-    yield "rg-idem-inttest-" + "".join(
+    yield "idem-rg-inttest-" + "".join(
```

Comment:
why are you changing the naming convention?
https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/naming-and-tagging